pkg/volume/csi/fake/fake_client.go

```diff
@@ -195,6 +195,29 @@ func (f *NodeClient) NodeUnstageVolume(ctx context.Context, req *csipb.NodeUnsta
 	return &csipb.NodeUnstageVolumeResponse{}, nil
 }
 
+// NodeExpandVolume implements csi method
+func (f *NodeClient) NodeExpandVolume(ctx context.Context, req *csipb.NodeExpandVolumeRequest, opts ...grpc.CallOption) (*csipb.NodeExpandVolumeResponse, error) {
+	if f.nextErr != nil {
+		return nil, f.nextErr
+	}
+
+	if req.GetVolumeId() == "" {
```

Comment:
not related to this pr, but these would be good validations to add to csi-sanity

Reply:
ack. I will create a github issue to track this in csi-test repo.